nl_open_data/flows/cbs/mlz.py

```diff
@@ -1,78 +1,143 @@
+        id=ids,
+        odata_version=odata_versions,
+        source=unmapped(source),
+        config=unmapped(config),
     )
-
-
-def main(config):
-    """Executes cbs.regionaal.flow in DaskExecutor.
-    """
-
-    """ Trigger in Prefect, load column description first and when finished only then load the data.
-    """
-    flow.set_reference_tasks([mlz_column_description])
-
-    # executor = DaskExecutor(n_workers=8)
-    flow.run(
-        # executor=executor,
-        parameters={"gcp": config.gcp,},
+    files_parquet = tables_to_parquet.map(
+        id=ids,
+        urls=urls,
+        odata_version=odata_versions,
+        source=unmapped(source),
+        pq_dir=pq_dir,
+    )
+    descriptions = get_dataset_description.map(urls=urls, odata_version=odata_versions)
+    desc_files = write_description_to_file.map(
+        description_text=descriptions,
+        id=ids,
+        pq_dir=pq_dir,
+        source=unmapped(source),
+        odata_version=odata_versions,
+        upstream_tasks=[descriptions],
+    )
+    col_descriptions = get_column_descriptions.map(
+        urls=urls, odata_version=odata_versions
+    )
+    col_desc_files = write_col_decription_to_file.map(
+        id=ids,
+        col_desc=col_descriptions,
+        pq_dir=pq_dir,
+        source=unmapped(source),
+        odata_version=odata_versions,
+        upstream_tasks=[col_descriptions],
+    )
+    gcs_folders = upload_to_gcs.map(
+        dir=pq_dir,
+        source=unmapped(source),
+        odata_version=odata_versions,
+        id=ids,
+        config=unmapped(config),
+        gcp_env=unmapped(gcp_env),
+        upstream_tasks=[files_parquet, desc_files, col_desc_files],
+    )
+    file_names = get_file_names.map(files_parquet)
+    dataset_refs = gcs_to_gbq.map(
+        id=ids,
+        source=unmapped(source),
+        odata_version=odata_versions,
+        third_party=unmapped(third_party),
+        config=unmapped(config),
+        gcs_folder=gcs_folders,
+        file_names=file_names,
+        gcp_env=unmapped(gcp_env),
+        upstream_tasks=[gcs_folders],
+    )
+    desc_dicts = get_col_descs_from_gcs.map(
+        id=ids,
+        source=unmapped(source),
+        odata_version=odata_versions,
+        config=unmapped(config),
+        gcp_env=unmapped(gcp_env),
+        gcs_folder=gcs_folders,
+        upstream_tasks=[gcs_folders],
+    )
+    bq_updates = bq_update_main_table_col_descriptions.map(
+        dataset_ref=dataset_refs,
+        descriptions=desc_dicts,
+        config=unmapped(config),
+        gcp_env=unmapped(gcp_env),
+        upstream_tasks=[desc_dicts],
     )
+    remove_dir.map(pq_dir, upstream_tasks=[gcs_folders])
 
 
 if __name__ == "__main__":
-    config = get_config("dataverbinders")
-    main(config=config)
+    from nl_open_data.config import get_config
+    from pathlib import Path
+    import os
+
+    config_file = Path.home() / Path("Projects/nl-open-data/nl_open_data/config.toml")
```

Comment:
this doesn't make sense to me. Isn't this still old code?

Reply:
Yes, sorry - I should have been clearer - only `test_statline_bq_flow.py` is ready, the rest are either completely old, or halfway through, and do not need a check yet. If we like the template, I will implement in different flows later.